docs/design/marketing-offer-program.md

```diff
@@ -67,54 +67,145 @@ The credit is automatically applied to the account and will have a max limit tha
     id - int
     name -  text
     description - text  
-    credits - integer
+    award_credits_in_cents - integer
+    invitee_credits_in_cents - integer
     redeemable_cap - integer
     num_redeemed - integer
     created_at - timestamp
     expires_at - timestamp
-    award_credit_duration_days - int
+    award_credit_duration_months - int
     invitee_credit_duration_days - int
-    type - enum[FREE_TIER, REFERRAL]
-    status - enum[ON_GOING, DEFAULT, EXPIRED, NO_STATUS]
+    status - enum[ACTIVE, DEFAULT, DONE]
```

Comment:
Could we offload this logic to the software and just store an integer for that?
I feel that is better performing and needs less resources

Reply:
That's the idea. I wrote it this way to show the design of the database